cms/envs/common.py

```diff
@@ -168,7 +168,7 @@
     },
 
     # Teams feature
-    'ENABLE_TEAMS': False,
+    'ENABLE_TEAMS': True,
```

Comment:
@dan-f these will not go in, i enabled here to avoid enabling them on sandbox manually.